.coveragerc

```diff
@@ -1,4 +1,4 @@
 [run]
 branch = true
-include = suitcase/tiff/*
-omit = suitcase/tiff/_version.py
+include = suitcase/tiff_series/*
```

Comment:
should this also cover / omit the stack version too?

Reply:
done